sorts/comb_sort.py

```diff
@@ -15,40 +20,43 @@
 
 def comb_sort(data):
     """Pure implementation of comb sort algorithm in Python
-    :param collection: some mutable ordered collection with heterogeneous
-    comparable items inside
-    :return: the same collection ordered by ascending
+    :param data: mutable collection with comparable items
+    :return: the same collection in ascending order
     Examples:
     >>> comb_sort([0, 5, 3, 2, 2])
     [0, 2, 2, 3, 5]
     >>> comb_sort([])
     []
-    >>> comb_sort([-2, -5, -45])
-    [-45, -5, -2]
+    >>> comb_sort([99,45,-7,8,2,0,-15,3])
+    [-15, -7, 0, 2, 3, 8, 45, 99]
     """
     shrink_factor = 1.3
     gap = len(data)
-    swapped = True
-    i = 0
+    completed = False
 
-    while gap > 1 or swapped:
-        # Update the gap value for a next comb
-        gap = int(float(gap) / shrink_factor)
+    while not completed:
 
-        swapped = False
-        i = 0
+        # Update the gap value for a next comb
+        gap = int(gap / shrink_factor)
```

Comment:
gap //= shrink_factor  # will give us integer division

Reply:
does not work, return type is still float

Reply:
You are correct.  Sorry for the mistake.
```python
>>> 35.2 / 7
5.028571428571429
>>> 35.2 // 7
5.0
```